### PR TITLE
DSP-45255 Add shading module for Jackson with fixed CVE version

### PR DESCRIPTION
**Pull request by diegomon**
DSP-45255 Add shading module for Jackson with fixed CVE version

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -47,8 +47,11 @@ stages:
     - sysctl -w net.ipv4.tcp_keepalive_time=60
     - sysctl -w net.ipv4.tcp_keepalive_intvl=60
     - sysctl -w net.ipv4.tcp_keepalive_probes=10
+
   script:
-    # upload artifacts
+    # Upload flink shaded jackson artifacts
+    - su -c "cd flink-shaded-jackson-parent && $MVN $MAVEN_CLI_COMMON_OPTS -Pdocs-and-source -DaltDeploymentRepository=releases.repo.splunk.com::default::https://repo.splunk.com/artifactory/maven-splunk-local $MAVEN_CLI_DEPLOY_OPTS $CUSTOM_JOB_OPTS deploy"
```
Comment (mxm) [inline .gitlab-ci.yml]: We also need to install this artifact before we build. Deploy always runs through a build.

Comment (diegomon) [reply .gitlab-ci.yml]: Thanks! I added back the install script

**Review by mxm — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (diegomon) [reply .gitlab-ci.yml]: After checking. It is not necessary because install is run as part of deploy

Comment (mxm) [reply .gitlab-ci.yml]: In any case, we must make sure that we run install before the main Flink build. The jackson project and the Flink project separate Maven projects, so we must make sure to run the Jackson project first.

**Pull request merged**